static/src/images/global/market-up.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" width="15" height="11" viewBox="0 0 15 11"><path fill="#298422" d="M0 9.97L1.03 11H14l1-1L8 0h-.958L0 9.984z"/></svg>
```

Comment:
I'd like to see proof, as you're basically swapping CPU cycles for IO, and I suspect the latter is more expensive on patchy mobile devices, especially when the HTML templates are small.

One issue is we don't use a proper templating library, but a helper that we've written, which quite likely doesn't perform anywhere near as well as something that's been tuned.